src/assets/gagesBarChartAnimation/gagesBarChartAnimationText.js

```diff
@@ -3,9 +3,5 @@ export default {
         title: "Gages Through the Ages ",
         introText: "A visual history of streamgaging in the US ",
         graphTitle: "",
-        caption:"U.S. Geological Survey’s active streamgages since 1889 until present day",
-        svgText: "The U.S. Geological Survey (USGS) Streamgaging Program is funded by both Congress and over 1,400 state and local partner agencies to monitor river water levels and flow, and to support water-quality monitoring across the U.S.",
-        paragraphText1: "In 2020, USGS monitors water levels and flow at more than 8,400 streamgages, some of which also are equipped with sensors that measure water properties such as temperature, conductivity, and dissolved oxygen content. Streamgages send data in near real-time to the <a href='https://waterdata.usgs.gov/nwis' target='_blank'>National Water Information System (NWIS)</a>. Streamgage data have become critical to forecasting floods, managing streamflow, and operating hydropower plants, as well as recreation, emergency management, engineering designs, and more.",
-        paragraphText2: "The USGS began using streamgages to quantify the Nation's flowing waters in 1889. Factors that influence which waterways are selected for monitoring have changed in the last 130 years. Gages are built at specific locations for many reasons, and those reasons are reflected in the number and location of gages through time. Their placement tells a story of changing needs for the Nation. "
-    }
```

Comment:
I think in addition to wanting these intro paragraphs (as you noted), we want that `}` on L10, too